packages/nextjs/components/scaffold-eth/Address/Address.tsx

```diff
@@ -102,12 +122,11 @@ export const Address = ({ address, disableAddressLink, format, size, onlyEnsOrAd
   const displayAddress = format === "long" ? checkSumAddress : shortAddress;
   const displayEnsOrAddress = ens || displayAddress;
 
-  size = size ?? (onlyEnsOrAddress || (!ens && !isEnsNameLoading) ? "base" : "xs");
-  const addressSize = size;
+  const showAddressWithEnsOrEnsSkeleton = !onlyEnsOrAddress && (ens || isEnsNameLoading);
 
-  const shouldTryEnsWithAddress = !onlyEnsOrAddress && (ens || isEnsNameLoading);
-  const blockieSize = shouldTryEnsWithAddress ? getNextSize(blockieSizeMap, size, 4) : size;
-  const ensSize = shouldTryEnsWithAddress ? getNextSize(textSizeMap, size) : size;
+  const addressSize = showAddressWithEnsOrEnsSkeleton ? getPrevSize(textSizeMap, size, 2) : size;
```

Comment:
to test first option https://github.com/scaffold-eth/scaffold-eth-2/pull/924#issuecomment-2356858635 change this line to `const addressSize = size;`